packages/datagateway-common/src/table/table.component.tsx

```diff
@@ -96,7 +97,11 @@ interface VirtualizedTableProps {
   loadMoreRows?: (offsetParams: IndexRange) => Promise<unknown>;
   totalRowCount?: number;
   sort: { [column: string]: Order };
-  onSort: (column: string, order: Order | null) => void;
+  onSort: (
+    column: string,
+    order: Order | null,
+    defaultSort: UpdateMethod
```

Comment:
Again, defaultSort sounds like the name of a bool - and not the strings "push" or "replace"

Reply:
Oops, yes I think I must have copy and pasted, it should have matched the `onSort` in index.tsx. Thanks for spotting that 😄